library/aes.c

```diff
@@ -605,13 +603,18 @@ int mbedtls_aes_setkey_enc(mbedtls_aes_context *ctx, const unsigned char *key,
 
 #if defined(MBEDTLS_AESCE_HAVE_CODE)
     if (MBEDTLS_AESCE_HAS_SUPPORT()) {
-        return mbedtls_aesce_setkey_enc((unsigned char *) RK, key, keybits);
```

Comment:
How come the compiler didn't catch this in the CI?

Reply:
As part of this PR, the function changed to be `void` (it's an internal function) as the checks were duplicated by the only caller.